src/yaml-support/yaml-schema.ts

```diff
@@ -116,6 +138,23 @@ class KubernetesSchemaHolder {
             this._definitions[schema.id.toLowerCase()] = schema;
         }
     }
+
+    // get the markdown format of document for the current property and the type of current property
+    // see similar function chaseFieldPath in expaliner.ts
```

Comment:
Would it make sense to try to unify this with `chaseFieldPath` or something at that level, so we don't need to duplicate the logic, or export the low-level details of `explainOne` and `explainComplex`?  Seems like we should be able to have the explainer module have just one or two high-level entry points and hide all the complexity.  But perhaps that's for a later refactoring.

Reply:
explain command will be refactored for explain the user selection text or user input later